Beeper/BarcelonaMautrixIPC/Commands/Structs/BLMessage.swift

```diff
@@ -92,6 +97,7 @@ public struct BLMessage: Codable, ChatResolvable {
         }
 
         chat_guid = message.blChatGUID ?? "unknown"
+        thread_id = message.groupID ?? ""
```

Comment:
We should remove the nil-coalescing here since the variable is optional
```suggestion
        thread_id = message.groupID
```